doc/cascadia/profiles.schema.json

```diff
@@ -602,6 +602,58 @@
         }
       ]
     },
+    "PrevTabAction": {
+      "description": "Arguments corresponding to a Previous Tab Action",
+      "allOf": [
+        { "$ref": "#/definitions/ShortcutAction" },
+        {
+          "properties": {
+            "action": { "type":"string", "pattern": "prevTab" },
+            "tabSwitcherMode": {
+              "oneOf": [
+                { "type": "null" },
+                {
+                  "enum": [
+                    "mru",
+                    "inOrder",
+                    "disabled"
+                  ],
+                  "type": "string"
+                }
```

Comment:
nit: we could probably pull this out to it's own definition, so we wouldn't have to duplicate it in both, but that's not a big deal.